src/dnx/pal.win32.cpp

```diff
@@ -81,19 +83,56 @@ BOOL GetFullPath(LPCTSTR szPath, LPTSTR pszNormalizedPath)
     return TRUE;
 }
 
+namespace
+{
+    std::wstring get_runtime_path(TraceWriter& trace_writer)
+    {
```

Comment:
The servicing index can be located in a different folder, other than program files, if you set the `DNX_SERVICING` env var. https://github.com/aspnet/dnx/blob/32358ec9b8970070ff77cbc5beef28c13e3ab09b/src/Microsoft.Framework.Runtime/Servicing/ServicingTable.cs#L31